src/transformers.js

```diff
@@ -73,8 +73,12 @@ export async function transform(
       return embed;
     })
   );
-  await replaceEmbedsInHtml(embedsWithResources, lang);
-  const embedMetaData = getEmbedMetaData(embedsWithResources, lang);
+  await replaceEmbedsInHtml(embedsWithResources, lang, options.path);
+  const embedMetaData = getEmbedMetaData(
+    embedsWithResources,
+    lang,
+    options.path
+  );
```

Comment:
Eg er usikker på om det er lurest å sende inn options eller options.path som parameter her. Resultatet er det samme, men det er kanskje litt reinere å sende inn objektet. Det gjør det i alle fall lettere om det seinere skulle bli behov for å sende inn fleire parametre enn path, for då slepp en å endre funksjonssignaturen

Reply:
Er det et poeng å ha forskjellig navn på parameteret? Det er jo options fra server.js som sendes inn uansett så kvifor ikkje berre kalle det det i stedet for metaOptions eller htmlOptions.